examples/petsc/bpsraw.c

```diff
@@ -35,7 +35,7 @@
 //     ./bpsraw -problem bp5 -ceed /omp/occa
 //     ./bpsraw -problem bp6 -ceed /ocl/occa
 //
-//TESTARGS -ceed {ceed_resource} -test -problem bp2 -degree 5 -qextra 5
+//TESTARGS -ceed {ceed_resource} -test -problem bp2 -degree 4 -qextra 5
```

Comment:
The issue wasn't that `p == q` or `q > p` but that the problem was too large for `/gpu/cuda/ref`.

Reply:
Dropping this drops an AVX case.

Reply:
I'm dropping this commit for now since it appears that something deeper is going on with /cuda/ref and I don't want to reduce the coverage of the AVX backends.